ocaml/xapi/xapi_globs.ml

```diff
@@ -1834,6 +1836,11 @@ let other_options =
     , (fun () -> string_of_int !max_span_depth)
     , "The maximum depth to which spans are recorded in a trace in Tracing"
     )
+  ; ( "https-only"
+    , Arg.Set https_only
+    , (fun () -> string_of_bool !https_only)
+    , "Only service HTTPS request, disable HTTP/80 when set to true"
```

Comment:
IIUC this only applies to newly created hosts, setting this to true won't turn HTTP off for existing hosts.
Or maybe I'm misunderstanding how this works.
We should make that explicit in the name of this configuration field to avoid confusion.
Maybe `create-https-only`, and in the docstring say "Create new hosts in https-only mode, disabling HTTP/80. Existing hosts and the current host are unaffected".